styleguide.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+
+<head>
+	<meta charset=utf-8>
+	<title>Styleguide - cafébebel 2017</title>
+	<link rel="stylesheet" type="text/css" href="./styleguide.css"/>
+</head>
+
+<h1>h1: Chewie, we're home.</h1>
+
+<h2>h2: They call it a Royale with cheese.</h2>
+
+<h3>h3: Why so serious?</h3>
+
+<h4>h4: Mama says, 'Stupid is as stupid does.</h4>
+
+<h5>h5: Elementary, my dear Watson.</h5>
+
+<h6>h6: Magic Mirror on the wall, who is the fairest one of all?</h6>
+
+
+<p>p: My name is Maximus Decimus Meridius, commander of the Armies of the North, General of the Felix Legions and loyal servant to the true emperor, Marcus Aurelius. Father to a murdered son, husband to a murdered wife. And I will have my vengeance, in this life or the next.</p>
+
+<p>p a: The first rule of Fight Club is: You do not talk about Fight Club. The second rule of <a href=#>Fight Club</a> is: You do not talk about Fight Club.</p>
+
+<blockquote>blockquote: "Say hello to my little friend!"</blockquote>
+
+<ul>
+	<li>li : Pierre</li>
+	<li>li : Pierre</li>
+	<li>li : Pierre</li>
+</ul>
+
+<form>
+  <input type="text" name="firstname" placeholder="First Name">
+  <input type="text" name="lastname" placeholder="Last Name">
+  <input type="email" name="email" placeholder="Email">
+  <input type="submit" value="Sign in">
```

Comment:
Same here, quotes are only required for `value` because of the space.
This applies to all other fields.